### DISCUSSION TITLE
restyle login screen widths to support long words

### DISCUSSION THREAD

**Discussion by olets** [craftcms/cms#8563] [Ideas]
### Description

On the log-in screen, long single-word site names break across lines. It would be easy to adjust styles to support long words without negatively affecting the appearance of the rest of the page (for example, the 300px width could wrap the form but not the site name).

![screen shot 2018-07-10 at 12 53 58](https://user-images.githubusercontent.com/3282350/42525120-5ae8207a-8440-11e8-82c9-818dac8d28f2.png)

### Steps to reproduce

Create a Craft site with a long site name (e.g. `mylongwordsitename`) and then visit `/admin` (or go to the login screen and use Inspector to replace your site name with `mylongwordsitename`)

### Additional info

- Craft version: 3.0.12

**Comment by narration-sd** [comment]
@olets This might be a little trickier to solve, while I appreciate why you'd like it so -- particularly after living long in a German language environment, though hyphenated personal names can have the same issues.

I'd guess the 300px fixed width is so that the Craft CP will work conveniently on narrower cellphone/handy screens, as the not quite current  generation often were 360px.

So a solution would probably be to lower the font size in such cases, probably taking some JavaScript, unless Brandon's wizardry may come up with something more clever, as oft :)
